1-js/06-advanced-functions/04-var/article.md

```diff
@@ -169,16 +169,16 @@ function sayHi() {
 sayHi();
 ```
 
-Because all `var` declarations are processed at the function start, we can reference them at any place. But variables are undefined until the assignments.
+Because all `var` declarations are processed at the function start, we can reference them at any place. But variables are undefined until they are explicitly defined.
 
 In both examples above `alert` runs without an error, because the variable `phrase` exists. But its value is not yet assigned, so it shows `undefined`.
 
 ## Summary
 
 There are two main differences of `var`:
 
-1. Variables have no block scope, they are visible minimum at the function level.
-2. Variable declarations are processed at function start.
+1. `var` Variables have no block scope, they are not block-local and can be accessed from any point in the script.
```

Comment:
wrong, they are function-local, so can't be "accessed from any point in the script"  =)

Reply:
not sure if rephrasing is good here...

Reply:
LGTM